File: Easy1/Easy1.py
#Python Easy1
print("What is your name, age, and username?")
NameInput = input("Name:")
AgeInput = input("Age:")
UserNameInput = input("Username:")
print('Your name is {}, you are {} years old, and your username is {}.'.format(NameInput, AgeInput, UserNameInput))
f = open('pyRedditBioList', 'w')
f.write('{} {} {}'.format(NameInput, AgeInput, UserNameInput))

File: Easy1/Easy1.c
#include <stdio.h>
int main()
{
	char Name [80];
  char Username [80];
	int age;
	
	printf("Enter your name:");
	scanf("%s", Name);
	printf("Enter your age:");
	scanf("%d", &age);
	printf("Enter your Username:");
	scanf("%s", Username);
	
	printf("Your name is %s, you are %d years old, and your username is %s", Name, age, Username);
	FILE *fp;
	fp = fopen("cRedditBioList.txt", "w");
	fputs(Name, fp);
	fprintf(fp, " %d ", age);
	fputs(Username, fp);
	return 0;
}